src/transformers/trainer_utils.py

```diff
@@ -156,36 +156,34 @@ class EvalPrediction:
     Parameters:
         predictions (`np.ndarray`): Predictions of the model.
         label_ids (`np.ndarray`): Targets to be matched.
-        inputs (`np.ndarray`, *optional*):
+        inputs (`np.ndarray`, *optional*): Input data passed to the model.
+        losses (`np.ndarray`, *optional*): Loss values computed during evaluation.
```

Comment:
Should be marked as optional in the docstring

Reply:
Thank you @amyeroberts for the review!

my bad I had commented yesterday under your review and just realized I had a "pending" label so the comment was only visible on my end.

I’ve already specified the arguments in the docstring as optional, are you suggesting that I should add (Optional[np.ndarray], optional) or include the phrase defaults to None ?

Reply:
Sorry, my bad, I must have just skimmed and not read properly. All good!